main/guides/js-programming/agoric-js-overview.md

```diff
@@ -198,7 +196,7 @@ sneaky = import
 // tricky comment to obscure function invocation
```

Comment:
This is an example where our `import` regexp successfully rejects code that should be rejected, but the text suggests the opposite. We need to make clear that for this prohibition (unlike direct eval) we *only* make mistakes of falsely rejecting code that was actually safe. We believe we never mistakenly accept code that we should have rejected. This example looks like it is an example of the second.